src/render/extract.rs

```diff
@@ -120,14 +121,11 @@ pub fn extract(
         Query<
             Entity,
             Or<(
-                Added<TilemapMeshType>,
```

Comment:
A lack of an `Added` query filter here seems strange to me as we wont detect when a tile map initially spawns?

Reply:
@StarArawn I guess we should have an `Added<TilemapStorage>` here...